### DISCUSSION TITLE
Email notifications

### DISCUSSION THREAD

**Discussion by Captshield** [likehopper/redmine_sla#12] [Ideas]
Hi,

Based on the description, it's an excellent plugin. 
One thing I miss is to be able to send email notifications, e.g. in the following cases (as far as I can see, there are no such notifications yet):

- when SLA is breached
- with a specified time before the violation (e.g. in 2h 30m)
- with the % given before the violation (e.g. 80%)

repetition at specified intervals (e.g. 1d)

Several of these could be configured for each SLA type.
The recipient could be specified (Assignee, Reporter, Watchers, Given Roles, Given users), several of these.

This would help the team to avoid any SLA violation.

Thanks,
Peter

**Comment by likehopper** [comment]
Thank you for this feedback, I'm starting to think about how to manage notifications as best as possible.
